src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services ComputerVision APIs.</Description>
-    <Version>3.4.0</Version>
+    <Version>4.0.0</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;REST HTTP client;Computer Vision;Computer Vision API;Computer Vision SDK;Vision;netcore451511</PackageTags>
     <PackageReleaseNotes>
```

Comment:
Please update the `PackageReleaseNotes` here

Reply:
Updated.